newsfragments/2860.misc.rst

```diff
@@ -0,0 +1 @@
+Moved the metadata into ``PEP 621``-compliant ``pyproject.toml``.
```

Comment:
Nit: could use Sphinx-specific roles here.
```suggestion
Moved the metadata into :pep:`PEP 621`-compliant :file:`pyproject.toml`.
```

Reply:
You should remove the word `PEP ` from between the backticks (not sure if it work with that inside).